compiler/shared/test/diff-ir/IR.mls

```diff
@@ -14,28 +15,9 @@ foo()
 //│ 
 //│ 
```

Comment:
Is there any reason for these two empty lines? One might be enough.

Reply:
none that I can see, changed to one line